### PR TITLE
Change example project more simply

### PR DESCRIPTION
**Pull request by munisystem**
## WHY
To explain apig usage.

## WHAT
Add apig sample project considering model association.

### CODE REVIEW

**Comment by dtan4** [comment]
`_example/apig` should be removed!

File: _example/controllers/user.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
+package controllers
+
+import (
+	"net/http"
+
+	dbpkg "github.com/wantedly/_example/db"
+	"github.com/wantedly/_example/helper"
+	"github.com/wantedly/_example/models"
+	"github.com/wantedly/_example/version"
```
Comment (dtan4) [inline _example/controllers/user.go]: `apig` is missing! `github.com/wantedly/apig/_example/*`

Comment (munisystem) [inline _example/controllers/user.go]: Oops... Fixed.

**Comment by dtan4** [comment]
We don't have `.travis.yml` now (preparing at #42 ), so these CI errors can be ignored.

**Comment by munisystem** [comment]
```
>>> cd $GOPATH/src/github.com/wantedly/apig
>>> git pull
remote: Counting objects: 16, done.
remote: Compressing objects: 100% (15/15), done.
remote: Total 16 (delta 2), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (16/16), done.
From https://github.com/wantedly/apig
   54b6337..cf48891  munisystem/example-project -> origin/munisystem/example-project
   3114bf0..8d38055  master     -> origin/master
Updating 54b6337..cf48891
Fast-forward
 _example/README.md            |  20 +++++-----
 _example/controllers/.gitkeep |   0
 _example/db/db.go             |   2 +-
 _example/docs/index.apib      |   2 +-
 _example/docs/job.apib        |   6 +--
 _example/docs/user.apib       |   6 +--
 _example/helper/field.go      | 114 ++++++++++++++++++++++++++++++++++--------------------
 _example/helper/field_test.go | 131 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 _example/router/router.go     |  12 +++---
 9 files changed, 227 insertions(+), 66 deletions(-)
 create mode 100644 _example/controllers/.gitkeep
 create mode 100644 _example/helper/field_test.go
>>> cd _example
>>> go build
>>>
```
Build success.

**Comment by dtan4** [comment]
LGTM ❗

**Pull request merged**